packages/ui/src/components/va-skeleton/VaSkeleton.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <component
+    class="va-skeleton"
+    role="status"
+    aria-busy="true"
```

Comment:
If VaSkeleton is show via `v-if`, `aria-busy` and `aria-atomic` are redundant imo.

Reply:
It shows with some delay, so if you just refresh page it doesn't blink and shows only if something loading for a very long time (>300ms), but still yet, the component itself represent loading and if page is loading 700ms, skeleton will be shown with busy state for a 400ms.

Reply:
I think it might be better to render this element, so it is busy anytime even before delay. We can visually hide it so it is no blinking, but still present in DOM as loading indicator. That's makes sense for me.